tests/example/longest_path/longest_path.dl

```diff
@@ -0,0 +1,16 @@
+// Finds the length of the path that visits the most nodes without overlapping itself
```

Comment:
Needs the standard license header. e.g. https://github.com/souffle-lang/souffle/blob/master/tests/semantic/load7/load7.dl